index.html

```diff
@@ -82,5 +82,8 @@ <h2 class="p-10">
             <a href="https://www.linkedin.com/in/kevin-hinds-49032b21b/">https://www.linkedin.com/in/kevin-hinds-49032b21b/</a>
         </div>
     </section>
+    <footer class="bg-black text-white flex p-10 justify-center">
+        <a class="text-white" href="">Back to top</a>
+    </footer>
```

Comment:
Consider adding more padding ya jabroni jk ... 
```suggestion
    <footer class="bg-black text-white flex p-20 justify-center">
        <a class="text-white" href="">Back to top</a>
    </footer>
```